spec/models/package_manager/go_spec.rb

```diff
@@ -44,6 +44,22 @@
     end
   end
 
+  describe "#project" do
+    it "returns raw project data from pkg.go.dev" do
+      VCR.use_cassette("go/pkg_go_dev") do
```

Comment:
🎱 Would be nice if only the relevant snippet was used here so it's easier to see what's being tested, vs replaying the entire request

Reply:
the `project()` method returns raw data from upstream, so I think we need the whole HTTP request to test that it's working.